### PR TITLE
Add cython-fortran-file

### PR DESCRIPTION
**Pull request by cphyc**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ngoldbaum** [comment]
Yay, looks like this is working. I believe you can ignore the appveyor build; the build passes on windows azure pipelines.

File: recipes/cython-fortran-file/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 9f54b216c0eb0b5a9a61e2305b093c052f1da2e9260e4566a1b4adb0b09d43a5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy
```
Comment (isuruf) [inline recipes/cython-fortran-file/meta.yaml]: Remove this line

File: recipes/cython-fortran-file/bld.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install     # Python command to install the script.
```
Comment (isuruf) [inline recipes/cython-fortran-file/bld.sh]: You can remove this file

File: recipes/cython-fortran-file/bld.bat [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (isuruf) [inline recipes/cython-fortran-file/bld.bat]: Same here

**Pull request merged**